source/style-guide/style/text-formatting.txt

```diff
@@ -43,7 +43,9 @@ elements. The following style differences are highlighted:
    Application names
      Apply no additional formatting.
 
-     .. seealso:: :ref:`product-names-version-numbers`
+     .. seealso:: 
+       
+        :ref:`product-names-version-numbers`
```

Comment:
Just to match the other entries:

```suggestion
        :ref:`product-names-version-numbers`.
```

Reply:
@sarahsimpers could you make the above change when you get a chance, thanks!

Reply:
Ah sorry, I didn't realize you were waiting. I'll ask the other reviewers to take a look as soon as they get the chance.

Per our style guide team rep process, we'll merge and deploy once everything is LGTM-ed.